src/variableProcessor.ts

```diff
@@ -4,48 +4,14 @@ import { commands, Uri, window } from 'vscode';
 import { EnvironmentController } from './controllers/environmentController';
 import * as Constants from './constants';
 import { Func } from './common/delegates';
+import * as adal from 'adal-node';
+import * as copyPaste from 'copy-paste';

```

Comment:
keep `copy-paste` and `node-uuid` modules uses the require syntax, since we don't add their typing files explicitly in `package.json`, so no intellisense will have from Vscode.

I encourage to use typescript module import syntax for `adal-node` is that, this package has typings file in its package directly.